src/main/java/kenkron/antiqueatlasoverlay/AAOConfig.java

```diff
@@ -1,76 +1,72 @@
 package kenkron.antiqueatlasoverlay;
 
-import blue.endless.jankson.Comment;
-import com.mojang.datafixers.kinds.App;
 import hunternif.mc.atlas.client.gui.GuiAtlas;
-import io.github.cottonmc.cotton.config.ConfigManager;
-import io.github.cottonmc.cotton.config.annotations.ConfigFile;
 
 public class AAOConfig {
 
     public static Position position = new Position();
     public static Appearance appearance = new Appearance();
 
     public static void loadConfig() {
-        position = ConfigManager.loadConfig(Position.class);
-        appearance = ConfigManager.loadConfig(Appearance.class);
+        //position = ConfigManager.loadConfig(Position.class);
+        //appearance = ConfigManager.loadConfig(Appearance.class);
```

Comment:
TODO: check that Overlay config values get updated in the game, and delete the comments.